jbmc/regression/jbmc-strings/StringBuilderAppend03/test1.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+test.class
+--function test.nondet --max-nondet-string-length 1000 --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar` --property "java::test.nondet:(Ljava/lang/String;)V.assertion.1"
+^EXIT=10$
+^SIGNAL=0$
+\[.*assertion\.1\] .* line 9 .* FAILURE
```

Comment:
What about the other two assertions?

Reply:
I'm testing per-property, so the other assertions are checked in the two other desc files. Is that what you mean?